### PR TITLE
Inline ads

### PR DESCRIPTION
**Pull request by mehdignu**
Fixes #
(comma separated list of tickets/issues/bugs fixed by the PR)

## Proposed Changes

  - add Ads between the paragraphs
https://sjira.funkemedien.de/browse/FDL-677

License: AGPLv3

### CODE REVIEW

File: src/SWP/Bundle/CoreBundle/Twig/ArticleBodyExtension.php [hunk 1]
```diff
@@ -49,9 +49,69 @@ public function getFunctions(): array
...
         return [
             new TwigFunction('setRendition', [$this, 'setRendition']),
+            new TwigFunction('setInlineAds', [$this, 'setInlineAds']),
         ];
     }
 
+    public function setInlineAds(string $body): array
+    {
+        # split the body into paragraphs
+        $paragraphs = explode("</p>", $body);
```
Comment (jhalitschke) [inline src/SWP/Bundle/CoreBundle/Twig/ArticleBodyExtension.php:59]: Don't we loose the closing p-tags?

`explode("</p>", '<h1>headline</h1><p>paragraph one.</p><p>paragraphtwo.</p>');`

Results in
```
Array
(
    [0] => <h1>headline</h1><p>paragraph one.
    [1] => <p>paragraphtwo.
    [2] => 
)
```

**Review by jhalitschke — CHANGES_REQUESTED** [review]
How to handle the closing p-tags?

File: src/SWP/Bundle/CoreBundle/Twig/ArticleBodyExtension.php (hunk 1 above)
Comment (mehdignu) [reply src/SWP/Bundle/CoreBundle/Twig/ArticleBodyExtension.php:59]: i've added the delimiter back. but this also handled in the template using the raw filter.

**Review by jhalitschke — APPROVED** [review]
ok

**Pull request merged**